### DISCUSSION TITLE
Single task

### DISCUSSION THREAD

**Discussion by mmmcorpsvit** [agronholm/apscheduler#926] [Q&A]
hello, i search way to migrate from Celery to dramatic + apscheduler
can help my please, how schedule tasks to run every 15 min (task can work few hours, I don't know how long), but need only one instance
can share any example please how this implement

**Comment by agronholm** [comment]
Start with the [official documentation](https://apscheduler.readthedocs.io/en/3.x/), and if you can't figure it out, let me know. But I want you to try figuring it out yourself first.

**Comment by mmmcorpsvit** [comment]
```python
@scheduler.scheduled_job('interval', seconds=60, **max_instances=1**)
def parser_copart(rows: int = 0, skiprows=0):
    pass
```

so easy, I miss this, very very sorry, my fault(((
thanks for cool library, good luck!

**Reply by agronholm** [reply]
My advice is to not use `@scheduled_job`, as it has been removed in the 4.0 pre-release. It was a design error. But if you don't need to upgrade, and it works well for you, then there's no problem.

**Reply by mmmcorpsvit** [reply]
Right now, I'm using version 3.10.4, but I'm okay with it. I'm a newbie to using your library, and maybe I'll try the new version later.

Anyway, I have a plan to adapt: [fastapi-scheduler](https://github.com/amisadmin/fastapi-scheduler) when you release it.

I use FastAPI and the [simple example from fastapi-scheduler](https://github.com/amisadmin/fastapi-scheduler#simple-example). If I understand correctly, I need to create a new Docker service (like a Celery Worker) and add some environment parameters to check if it's the main application or a worker container. Is this the correct strategy?

answer from chat gpt:
To answer your question: Yes, creating a new Docker service for the scheduler, similar to how you would set up a Celery Worker, is a correct strategy. You can differentiate between the main application and the worker container using environment variables or command-line arguments. This way, you can configure each container appropriately based on its role.

**Reply by agronholm** [reply]
I can't really comment on that, as this is the first time I've heard of that project.
